app/src/main/java/com/jerboa/ui/components/drawer/Drawer.kt

```diff
@@ -439,16 +440,11 @@ fun AvatarAndAccountName(
             }
         }
         Column {
-            PersonName(
-                name = myPerson?.display_name ?: account.name,
-                color = MaterialTheme.colorScheme.onSurface,
-            )
-            Text(
-                text = if (account.isAnon()) "" else "${account.name}@${account.instance}",
-                color = MaterialTheme.colorScheme.tertiary,
-                style = MaterialTheme.typography.labelSmall,
-                overflow = TextOverflow.Ellipsis,
-                maxLines = 2,
+            ItemAndInstanceTitle(
```

Comment:
I kinda prefer how it looked before, Display name with the name@instance below it

Reply:
K updated to something simpler.